lib/tenstorrent/bh_arc/telemetry_internal.c

```diff
@@ -4,16 +4,27 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <zephyr/kernel.h>
-
 #include "avs.h"
 #include "telemetry_internal.h"
-#include "pvt.h"
 #include "regulator.h"
 
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/sensor/tenstorrent/pvt_tt_bh.h>
+
 static int64_t last_update_time;
 static TelemetryInternalData internal_data;
 
+#ifdef CONFIG_DT_HAS_TENSTORRENT_BH_PVT_ENABLED
```

Comment:
Same - let's avoid `#ifdef`s if possible.

`__maybe_unused`, `COND_CODE_1()`, and `IS_ENABLED()` could probably be used.